MiniApp/Classes/core/Assets/console.js

```diff
@@ -0,0 +1,22 @@
+function log(emoji, type, args) {
+  window.webkit.messageHandlers.logging.postMessage(
+    `${emoji} console.${type}: ${Object.values(args)
+      .map(v => typeof(v) === "undefined" ? "undefined" : typeof(v) === "object" ? JSON.stringify(v) : v.toString())
+      .map(v => v.substring(0, 3000)) // Limit msg to 3000 chars
+      .join(", ")}`
+  )
+}
+
+let originalLog = console.log
+let originalWarn = console.warn
+let originalError = console.error
+let originalDebug = console.debug
+
+console.log = function() { log("📗", "log", arguments); originalLog.apply(null, arguments) }
+console.warn = function() { log("📙", "warning", arguments); originalWarn.apply(null, arguments) }
+console.error = function() { log("📕", "error", arguments); originalError.apply(null, arguments) }
+console.debug = function() { log("📘", "debug", arguments); originalDebug.apply(null, arguments) }
```

Comment:
I think this should be moved to the `js-miniapp-bridge` code so that it can be covered by unit tests. The `ios-bridge.ts` file is only included for iOS, so you can add it there without affecting Android:
https://github.com/rakutentech/js-miniapp/blob/master/js-miniapp-bridge/src/ios/ios-bridge.ts